README.md

```diff
@@ -58,7 +58,7 @@ Users can specify a different location for the file by using the `--path-to-run-
 node src/cli.js --path-to-run-logs path/to/file.json
 ```
 
-If a `from-date` is provided as an option when running the ICARE Extraction Client, it will be used to filter out any data elements that are recorded before that date. If a `to-date` is provided as an option, it will be used to filter out any data elements that are recorded after that date. If no `to-date` is provided, the default is today. If no `from-date` is provided, the ICARE Extraction Client will look to the run log file to find the most recent run and use the `to-date` of that run as the `from-date` for the current run, allowing users to only run the extraction on data elements that were not included in previous runs. If there are no previous run times logged, a `from-date` needs to be provided when running the extraction.
+If a `from-date` is provided as an option when running the ICARE Extraction Client, it will be used to filter out any data elements that are recorded before that date. If a `to-date` is provided as an option, it will be used to filter out any data elements that are recorded after that date. If no `to-date` is provided, the default is today. If no `from-date` is provided, the ICARE Extraction Client will look to the run log file to find the most recent run and use the `to-date` of that run as the `from-date` for the current run, allowing users to only run the extraction on data elements that were not included in previous runs. If there are no previous run times logged, a `from-date` needs to be provided when running the extraction, unless the `all-entries` option is provided. If the `all-entries` option is provided, the `from-date` and `to-date` options will be ignored and none of the data elements will be filtered by date.
```

Comment:
Do you think we should also mention here that when you use the `all-entries` option, no dates will be logged in the run log file?

Reply:
Good catch, updated.